kubecost-cloud-gcp-marketplace-license.md

```diff
@@ -0,0 +1,30 @@
+# Kubecost Cloud GCP Marketplace Licensing
+
+Kubecost Cloud is [available for licensing on GCP Marketplace](https://console.cloud.google.com/marketplace/product/kubecost-public/kubecost-cloud) and can be installed in minutes. This guide will take you through licensing through GCP Marketplace, and next steps for setting up Kubecost Cloud.
+
+Kubecost currently offers 30 days of Kubecost Cloud free without licensing fees.
+
+## Prerequisites
+
+* Set up a Google Cloud account with an attached [billing account](https://cloud.google.com/billing/docs/how-to/create-billing-account).
+* [Set up a Kubecost Cloud account](kubecost-cloud/cloud-installation-and-onboarding.md#creating-a-user-account)
+
+## GCP Marketplace licensing guide
+
+On the [Product details page for Kubecost](https://console.cloud.google.com/marketplace/product/kubecost-public/kubecost-cloud), select _Subscribe_. You will be taken to an Order Summary page.
```

Comment:
@bstuder99 think some images could be useful here?

Reply:
I've updated the doc